setuptools/command/editable_wheel.py

```diff
@@ -81,14 +93,51 @@ def _create_wheel_file(self, bdist_wheel):
         with TemporaryDirectory(suffix=archive_name) as tmp:
             tmp_dist_info = Path(tmp, Path(self.dist_info_dir).name)
             shutil.copytree(self.dist_info_dir, tmp_dist_info)
-            pth = Path(tmp, f"_editable.{editable_name}.pth")
-            pth.write_text(f"{_normalize_path(self.target)}\n", encoding="utf-8")
-
+            self._install_namespaces(tmp, editable_name)
+            self._populate_wheel(editable_name, tmp)
             with WheelFile(wheel_path, "w") as wf:
                 wf.write_files(tmp)
 
         return wheel_path
 
+    def _best_strategy(self):
```

Comment:
This was just a very early "prototype" of attempting of implementing different strategies for the editable install...